applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_pw_element.cpp

```diff
@@ -0,0 +1,1334 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
```

Comment:
Most tests in this file are based on the ones in `test_transient_Pw_element`, which you added :)
```suggestion
//  Main authors:  Gennady Markelov  
//                          Richard Faasse
```

Reply:
Changed. However, I am not sure that anybody outside our team pays attention. ;)

Reply:
That's indeed true, might be nice for a team discussion (I mean we have git, so it's probably not necessary to always have to add/adjust this).